pegasus/test/fixtures/sites/code.org/public/test_no_erb_in_yaml.md

```diff
@@ -0,0 +1,6 @@
+---
+title: <%= (1..3).to_a.join(',').inspect %>
+theme: with_title
+---
+
+# Hello
```

Comment:
Maybe worth a comment mentioning that this is used in a test, and which test that is?